internal/prometheusbp/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package prometheusbp provides common code used by Baseplate.go regarding
```

Comment:
```suggestion
// Package prometheusbp provides common code used by baseplate.go regarding
```
:no_bell: is the official name all lowercase?

Reply:
it's not :)